File: cs/proxy/db/DbGiftBagOperation.cs
/// <summary>
/// 
/// </summary>
namespace org.zgl.service.client.db{
	[Proxy(gameId = -7)]
	public interface DbGiftBagOperation : IHttpSyncService { 
		/// <summary>
		/// 打开成长礼包信息
		/// </summary>
		GiftBagInfoDto open(long arg0);
		/// <summary>
		/// 领取奖励
		/// </summary>
		ItemListDto receiveAward(long arg0);
	}
}

File: componet/src/main/java/org/zgl/service/client/db/DbGiftBagOperation.java
package org.zgl.service.client.db;

import org.zgl.dto.ItemListDto;
import org.zgl.dto.clinet.db.GiftBagInfoDto;
import org.zgl.proxy.MethodDesc;
import org.zgl.proxy.rule.IHttpSyncService;

/**
 * @author： 猪哥亮
 * @创建时间： 2018/8/24
 * @文件描述：
 */
public interface DbGiftBagOperation extends IHttpSyncService {
    //打开成长礼包 回发礼包信息
    @MethodDesc("打开成长礼包信息")
    GiftBagInfoDto open(long uid);
    //点击领取礼包奖励
    @MethodDesc("领取奖励")
    ItemListDto receiveAward(long uid);
}
